site/docs/components/badge/examples.mdx

```diff
@@ -36,14 +35,36 @@ Using a badge inline with text clarifies which part of the content the badge rel
 
 Specify the maximum displayed number with the `max` prop. If `value` exceeds `max`, it will clamp the value and append a (`+`) suffix.
 
-When you don't pass a max value to the component, the value will have a limitation of three digits by default, displaying 999+ for any values over 999.
+- Without a `max` value, the component limits the value to three digits by default, displaying `999+` for any values over 999.
 
 </LivePreview>
 <LivePreview componentName="badge" exampleName="BadgeWithString">
     
 ## Badge with string
 
-The `value` property can accept both a `number` and a `string`.
+The `value` property can accept both a `number` and a `string`. When you pass a string, the badge will not clamp the value.
+
+</LivePreview>
+
+<LivePreview componentName="badge" exampleName="DotBadge">
+
+## Dot badge
+
+A dot badge is a circular dot that notifies users of changes in state.
+
+### Best practices
+
+- Be consistent with the badge's position in a group of elements.
+- If there is a 3:1 contrast between the badge and the element it is attached to, the badge can be anchored to the element's contents.
+- If the contrast is less than 3:1, the badge should be anchored to the element.
+
```

Comment:
"If the icon graphic has number on the right top, avoid anchor the badge directly to it." @joshwooding what do you think?